.travis.yml

```diff
@@ -11,13 +11,22 @@ stages:
 #defaults
 language: node_js
 node_js: "node"
-
-install: npm install -g lerna --no-package-lock && npm run postinstall
+# `nvm install` happens before the cache is restored, which means
+# we must install our own npm elsewhere (`~/npm`)
+before_install: |
+  [[ ! -x ~/cached-bins/node_modules/.bin/npm ]] && {

```

Comment:
Look how with this new commit adding more comments to the script, since we have the modules already cached, this step is instant:

![image](https://user-images.githubusercontent.com/5030059/47375934-b829a700-d6f1-11e8-8c3f-8bacdc3840d2.png)